tests/svg.js

```diff
@@ -22,6 +22,9 @@ async function main() {
 
   await Promise.allSettled(
     files.map(async (file) => {
+      // Skip SVG checks if the extension isn't svg
+      if (!file.endsWith(".svg")) return null;
```

Comment:
Do you need to return anything? If not, try just `return;`

Reply:
Didn't even know that was possible 😅